src/fermilib/transforms/_test_bksf.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Tests  _bksf_test.py."""
+from __future__ import absolute_import
+import numpy as np
+
+import unittest
+
+from fermilib.ops import (FermionOperator,
+                          hermitian_conjugated,
+                          InteractionOperator,
+                          normal_ordered,
+                          number_operator)
+from fermilib.transforms import (get_interaction_operator,
+                                 reverse_jordan_wigner)
+from fermilib.transforms._jordan_wigner import (jordan_wigner,
+                                                jordan_wigner_one_body,
+                                                jordan_wigner_two_body)
+from _bksf import (bksf_edge_matrix,
+                                                bksf,
+                                                bksf_edge_operator_aij,
+                                                bksf_edge_operator_b,
+                                                bksf_generate_fermions,
+                                                bksf_vacuum,
+                                                bksf_number_operator)
```

Comment:
make sure to refactor the names here after your changes to bksf and double check pep8 on both files